node_modules/browser/web-viewer.js

```diff
@@ -134,7 +135,13 @@ const WebViewer = Component(({item, items}) => {
   onFocus: WebViewer.onFocus(item),
   // onAsyncScroll: WebViewer.onUnhandled,
   onClose: event => remove(items, x => x.equals(item)),
-  onOpenWindow: WebViewer.onUnhandled,
+  onOpenWindow: event => {
+    items.update(items => {
+      const item = open().merge({uri: event.detail.url,
+                                 isSelected: false});
```

Comment:
Nit: I think we should move `isSelected: false` into [initial form](https://github.com/mozilla/browser.html/blob/master/node_modules/browser/web-viewer/actions.js#L12-L28) and we could also modify [`open`](https://github.com/mozilla/browser.html/blob/master/node_modules/browser/web-viewer/actions.js#L30) to let pass in additional config & avoid more allocations.